cogs/admin.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, bot):
 
     @commands.command()
     @commands.check(repo.is_owner)
-    async def reload(self, ctx, name: str):
+    async def reload(self, ctx, name):
```

Comment:
I question your removal of `:str` in here.

Reply:
Sorry there was no notification for this? Anyway

Because the library handles command arguments as strings. It's not even library related it's just a string by default.

When you type
`!reload test`
This will be treated both by discord and discord.py as a string.
The library will "extract" the arguments so if you have
```python
@commands.command()
@commands.check(repo.is_owner)
async def reload(self, ctx, name):
```
The `name`  is **already** a string and in this case is `test`

When you call `str` converter:
`async def reload(self, ctx, name: str):`
You are basically telling the library to convert a string to a string.

Note that this is **not** type hinting , it is a specific feature of discord.py called converters.
It will try to convert the argument, in this case to string, but the argument is already a string thus making in obsolete and redundant.

It is literally the same as my pervious example
```python
msg = "Name"
argument = str(msg) # notice how here we convert a string to string? 
```

Reply:
Understandable, then it's just me who've been used to do `:str` before, however I still want it to be like such as it does no harm (afaik)

Reply:
Yes it won't cause any issues either way.
Np understandable, have a good day 👍